alembic/versions/e9513456df23_add_stats_tvl.py

```diff
@@ -1,24 +0,0 @@
-"""add stats_tvl
```

Comment:
This is the merge migration (indicates that this revision `e9513456df23` marks the merge point between  branched revisions `e951d7a7e751` and `05f17c77dd4d`).

This merge revision must not be deleted, but needs to be updated since you have deleted the `e951d7a7e751` empty revision (which was a child of `41805685552a`), line 14 should be (comment can also be updated to be consistent):

```python
down_revision = ('41805685552a', '05f17c77dd4d')
```

Can be confirmed that these are the correct branches that need to be merged, by running `alembic heads` which should return something like
```
05f17c77dd4d (head)
41805685552a (head)
```

In the end, `alembic history` should confirm that the merge point revision is correct:
```
41805685552a, 05f17c77dd4d -> e9513456df23 (head) (mergepoint), add stats_tvl
17810e991dcc -> 41805685552a, create stats_tvl table
17810e991dcc -> 05f17c77dd4d, Add position restake count field
e93e2e0034cb -> 17810e991dcc (branchpoint), Unique protocol bytes id
82db2e875c18 -> e93e2e0034cb, Create ProtocolPremium model
fe37d8daf410 -> 82db2e875c18, Create base Protocol model
<base> -> fe37d8daf410, Initial version
```

Reply:
Really appreciate the thorough explanation. Thanks man!

I think I got it right. The `alembic history` command returns what it should.

Reply:
Anytime man! Looks good now 🚀